all.json

```diff
@@ -1,6 +1,19 @@
 {
   "allow": [],
   "deny": [
+    "blockschainexplorer.com",
+    "blockschainssuppt.site",
+    "syncvalidatedapps.com",
+    "walletfix.weebly.com",
+    "nftswaps.live",
+    "swappynfts.online",
+    "natural-farm-bonus.online",
+    "outerring-bonus.online",
+    "sinceredoge-bonus.online",
+    "naturalfarm-bonus.online",
+    "62.197.136.191",
```

Comment:
@michalisFr  @dubstard This I'm assuming is ok? (I guess it is is a known bad block as well, but will block future legit sites here)

Reply:
Which one I should remove?

Reply:
Not suggesting removing, just asking (out of interest) on the IP.